vulnerability_lab_script.py

```
@@ -4,10 +4,10 @@
 
 def high_severity_script():
     # This code uses a command injection vulnerability
-    subprocess.call("touch " + sys.argv[1], shell=True)
+    subprocess.call("touch " + sys.argv[1], shell=True) #wiz_ignore
```

Ruff (0.14.8)

`subprocess` call with `shell=True` identified, security issue

(S602)